libs/api/domains/cms/src/lib/generated/contentfulTypes.d.ts

```diff
@@ -18,6 +18,9 @@ export interface IAboutSubPageFields {
 
   /** Content */
   content: Document
+
```

Comment:
Should this file be tracked?

cc. @jeremybarbet

Reply:
We talked about it with Sindri last night, and it's safer to keep the file on the repository. Removing it would create a huge dependency on contentful and in case contentful is down for some reason it would break all builds on our pipeline